GVFS/GVFS.Common/Git/DiffTreeResult.cs

```diff
@@ -75,7 +75,8 @@ public static DiffTreeResult ParseFromDiffTreeLine(string line, string repoRoot)
             // Take the mode, sha, operation part and split on a space then add the paths that were split on a tab to the end
             parts = parts[0].Split(' ').Concat(parts.Skip(1)).ToArray();
 
-            if (parts.Length != 6)
+            if (parts.Length != 6 ||

```

Comment:
(hm.. not intentional here.. I had some extra stuff come along in my branch that was slightly ahead of yours).